packages/esm-service-queues-app/src/home.component.tsx

```diff
@@ -1,19 +1,27 @@
 import React from 'react';
-import { type ConfigObject, useConfig } from '@openmrs/esm-framework';
+import { useConfig, useFeatureFlag } from '@openmrs/esm-framework';
 import ActiveVisitsTable from './active-visits/active-visits-table.component';
 import ActiveVisitsTabs from './active-visits/active-visits-tab.component';
 import ClinicMetrics from './patient-queue-metrics/clinic-metrics.component';
 import PatientQueueHeader from './patient-queue-header/patient-queue-header.component';
+import { type ConfigObject } from './config-schema';
+import DefaultQueueTable from './queue-table/default-queue-table.component';
 
 const Home: React.FC = () => {
-  const config = useConfig<ConfigObject>();
-  const useQueueTableTabs: boolean = config.showQueueTableTab;
+  const { showQueueTableTab, useActiveVisitsTable } = useConfig<ConfigObject>();
+  const useActiveVisitsTableViaFeatureFlag = useFeatureFlag('active-visits-table');
 
   return (
     <>
       <PatientQueueHeader />
       <ClinicMetrics />
-      {useQueueTableTabs ? <ActiveVisitsTabs /> : <ActiveVisitsTable />}
+      {showQueueTableTab ? (
+        <ActiveVisitsTabs />
+      ) : useActiveVisitsTable || useActiveVisitsTableViaFeatureFlag ? (
+        <ActiveVisitsTable />
+      ) : (
+        <DefaultQueueTable />
+      )}
```

Comment:
As above, we just want to show the QueueTable, not toggle between ActiveVisitsTable and DefaultQueueTable.  We should also get rid of the feature flag.